package.json

```diff
@@ -34,7 +34,9 @@
     "bmutex": "0.1.4",
     "bsert": "0.0.6",
     "bstring": "^0.3.4",
-    "semver": "^5.5.1"
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
This could probably be a `devDependency` since its only required for `npm test` to work

Reply:
addressed